components/script/dom/webidls/ServiceWorkerContainer.webidl

```diff
@@ -6,9 +6,7 @@
 [Pref="dom.serviceworker.enabled", Exposed=(Window,Worker)]
 interface ServiceWorkerContainer : EventTarget {
   [Unforgeable] readonly attribute ServiceWorker? controller;
-  //[SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
-
-  [NewObject, Throws] ServiceWorkerRegistration register(USVString scriptURL, optional RegistrationOptions options);
+  [SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
```

Comment:
@creativcoder I am not sure how to get started with `resolve_native` and `reject_error`. How do I implement that in my code? (does it go in the [return](https://github.com/servo/servo/pull/13419/files/b4b7fa5003454846f15ce0cadecbc726697b06cf#diff-2c27b3453cfacbe80bdac4094a0ad0c9R68))